### PR TITLE
mycommitmessage

### PR DESCRIPTION
**Pull request by i-am-mel-dev**
Closes #

## Proposed Changes

  - did some changes on an atlas function which is super unique and specific
  -
  -

### CODE REVIEW

File: niutils/niutils.py [hunk 1]
```diff
@@ -142,38 +142,43 @@ def compute_metric(data, atlas, mask, metric='avg', invert=False):
...
-            parcels[m, :] = (np.percentile(dist, 75, axis=0) -
-                             np.percentile(dist, 25, axis=0))
-        elif metric == 'var':
-            dist = data[atlas == label]
-            parcels[m, :] = data[atlas == label].var(axis=0)
-
-    rank = compute_rank(parcels)
-    if invert:
         print(f'Invert {metric} rank')
+        print(f'Invert {metric} rank')
```
Comment (smoia) [inline niutils/niutils.py:161]: Hey! this slipped in but it's not necessary

File: niutils/niutils.py [hunk 2]
```diff
@@ -142,38 +142,43 @@ def compute_metric(data, atlas, mask, metric='avg', invert=False):
...
+        rank = 100 - rank
```
Comment (smoia) [inline niutils/niutils.py:162]:
```suggestion

```

**Review by smoia — CHANGES_REQUESTED** [review]
It starts great! Check the review!

File: niutils/niutils.py (hunk 2 above)
Comment (i-am-mel-dev) [reply niutils/niutils.py:162]: Thank you, editing now.

**Review by smoia — APPROVED** [review]
LGTM! :balloon:

**Pull request merged**